tests/Core/MessageBar/FluentMessageBarTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Bunit;
+using Xunit;
+
+namespace Microsoft.Fast.Components.FluentUI.Tests.MessageBar;
+
+public class FluentMessageBarTests : TestBase
+{
+    private const string FluentAnchorRazorJs = "./_content/Microsoft.Fast.Components.FluentUI/Components/Anchor/FluentAnchor.razor.js";
+
+    public FluentMessageBarTests()
+    {
+        TestContext.JSInterop.SetupModule(FluentAnchorRazorJs);
+    }
+    
+    [Fact]
+    public void FluentMessageBar_Default()
```

Comment:
That's awesome to have a first test. an you schedule another PR to add more tests?
We'll need it to avoid non-regression.

Reply:
Yes, will add more tests.